mat64/vector.go

```diff
@@ -324,7 +329,7 @@ func (v *Vector) reuseAs(r int) {
 
 func (v *Vector) isZero() bool {
 	// It must be the case that v.Dims() returns
-	// zeros in this case. See comment in Reset().
```

Comment:
Add this comment back, but fix it so that there is a comment there. The text here presumably came from what exists in Dense: https://github.com/gonum/matrix/blob/master/mat64/dense.go#L315